### ISSUE TITLE
Test suite doesn't work on tox 3?

### ISSUE THREAD

**Issue by and-semakin** [apihackers/tox-asdf#1]
First, thank you for the great library! It's very useful!

I have tox 3.14.1 installed and get following output when try to execute test suite:

```
$ tox
...
ImportError while importing test module '/home/br0ke/git/tox-asdf/tests/test_options.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_options.py:3: in <module>
    from tox.session import prepare, build_session
E   ImportError: cannot import name prepare
...
```

Full log here: https://gist.github.com/and-semakin/ef6573d9a36904ef5f6e07d34e46389f

**Comment by and-semakin** [comment]
By the way I get the same error with tox 2.9.1. Do I do something wrong? :thinking:

**Comment by and-semakin** [comment]
Ok, got why my version of `tox` doesn't count -- that's because tox install another tox inside of test session, and it always installs the latest one -- 3.14.1.

Tox guys refactored this module [here](https://github.com/tox-dev/tox/commit/984ce025ad8e75e323877fb5f34128abcfba950d#diff-fa162e4b845a9c35e2b680c199480dc6), it's version 3.8.0.

**Comment by and-semakin** [comment]
Any updates on this? @noirbizarre

**Comment by and-semakin** [comment]
Fixed in #3.
